frontend/src/metabase/notifications/modals/CreateOrEditQuestionAlertModal/CreateOrEditQuestionAlertModal.unit.spec.tsx

```diff
@@ -95,6 +123,19 @@ function setup({
   setupWebhookChannelsEndpoint(webhooksResult);
   setupUserRecipientsEndpoint({ users: [] });
 
+  /**
+   * Technically this is a dirty hack to avoid loading enterprise packages
+   * and rely on typings from enterprise package to mock the user:
+   * - import { UserWithApplicationPermissions } from "metabase-enterprise/application_permissions/types/user";
+   * - createMockUser({ ... } as UserWithApplicationPermissions);
+   *
+   * However the trade-off is worth it, because 4 LOC keeps the test clean,
+   * avoids loading unnecessary code or writing e2e tests for this.
+   */
+  jest
+    .spyOn(PLUGIN_APPLICATION_PERMISSIONS.selectors, "canAccessSettings")
+    .mockReturnValue(userCanAccessSettings);
```

Comment:
we have an established pattern for testing paid features that we shouldn't deviate from

https://www.notion.so/metabase/Testing-paid-features-49e03e1b2a3b41ffb4ebdc7616a93d90